requirements-test.txt

```diff
@@ -3,7 +3,7 @@ aiofiles==0.6.0
 redis==3.5.3
 
 # BMT + dependencies
-bmt-lite-2.1.0==1.0.2
+bmt-lite-2.1.0==2.1.1
```

Comment:
This causing an error when trying to lock requirements for testing. 

`ERROR: Cannot install -r /app/requirements-test.txt (line 23) and bmt-lite-2.1.0==2.1.1 because these package versions have conflicting dependencies.
ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies`

This is because binder 4.1.0 depends on bmt-lite<2.0.

Reply:
Thanks! I've upgraded binder.